Frontend development/Libraries.md

```diff
@@ -0,0 +1,22 @@
+## Styling
+
+* **Susy**
+  * Susy is an a responsive customizable layout toolkit for Sass.
+  * More info with code examples is available [here.](http://susy.oddbird.net/)
+* **Normalize.css**
+  * A small CSS file that provides better cross-browser consistency in the default styling of HTML elements. A modern, HTML5 ready alternative to traditional CSS reset.
```

Comment:
...to **the** traditional...

Reply:
'Susy is a responsive', not 'an a responsive...'